packages/components/src/components/brandshape/brandshape.ts

```diff
@@ -206,6 +260,82 @@ export default class SdBrandshape extends SolidElement {
       :host {
         @apply block;
       }
+
+      .container--stylized {
+        background-color: transparent;
+        padding: 40px 0;
+        position: relative;
+        z-index: 1;
+        border-width: 0;
+      }
+
+      .container--stylized::before {
+        background-color: var(--internal-color-background, white);
+        color: var(--internal-color, black);
+        border-color: var(--internal-border-color, black);
+        border-width: var(--internal-border-width, 0);
+        border-style: solid;
+        border-radius: 0 60px;
+        bottom: 0;
+        content: '';
+        left: 0;
+        margin-bottom: calc(21.256% / 2);
+        margin-top: calc(21.256% / 2);
+        position: absolute;
+        right: 0;
+        top: 0;
+        transform: skewY(-11deg);
+        z-index: -1;
+      }
+
+      .container--outline-primary::before {
+        --internal-color-background: none;
+        --internal-border-color: rgb(var(--sd-color-primary, 0 53 142));
+        --internal-border-width: 2px;
+      }
+
+      .container--outline-white::before {
+        --internal-color-background: none;
+        --internal-border-color: var(--sd-color-white, white);
+        --internal-border-width: 2px;
+      }
+
+      .image-wrapper {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: calc(100% - 42.512%);
+        margin-bottom: calc(21.256% / 2);
+        margin-top: calc(21.256% / 2);
+        z-index: 2;
+        overflow: hidden;
+        border-radius: 0 60px;
+        transform: skewY(-11deg);
+      }
+
+      slot[name='image']::slotted(img) {
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        transform: skewY(11deg);
+        transform-origin: top left;
+      }
+
+      /* Responsive border-radius */
+      @media (min-width: 414px) {
+        .container--stylized::before,
+        .image-wrapper {
+          border-radius: 0 72px;
+        }
+      }
+
+      @media (min-width: 640px) {
+        .container--stylized::before,
+        .image-wrapper {
+          border-radius: 0 84px;
+        }
+      }
```

Comment:
Could you use Tailwind for the CSS styles? Additionally, can you check if it might be possible to resolve the classes in the CSS and do more in the HTML?